presto-hive/src/test/sql/create-test.sql

```diff
@@ -232,3 +232,8 @@ ALTER TABLE presto_test_partition_schema_change REPLACE COLUMNS (t_data DOUBLE);
 
 INSERT OVERWRITE TABLE presto_test_partition_schema_change_non_canonical PARTITION (t_boolean='0')
 SELECT 'test' FROM presto_test_sequence LIMIT 100;
+
+ANALYZE TABLE presto_test_unpartitioned COMPUTE STATISTICS;
+ANALYZE TABLE presto_test_unpartitioned COMPUTE STATISTICS FOR COLUMNS;
+ANALYZE TABLE presto_test_bucketed_by_string_int PARTITION(ds) COMPUTE STATISTICS;
+ANALYZE TABLE presto_test_bucketed_by_string_int PARTITION(ds) COMPUTE STATISTICS FOR COLUMNS;
```

Comment:
>  I don't know of any reason why it would cause failures

On travis you have limit on memory, running analyze might require running a map reduce job (this is why it is slow) in hadoop, which spins at least one new jvm process which may cause proces ripper to kill some process.

Since your travis passes it looks like we are good.